sklearn/multioutput.py

```diff
@@ -565,11 +565,10 @@ class ClassifierChain(MetaEstimatorMixin, ClassifierMixin, _BaseChain):
 
         If order is 'random' a random ordering will be used.
 
-    cv : int, cross-validation generator or an iterable, optional \
-    (default=None)
+    cv : int, cross-validation generator or an iterable, default=None
```

Comment:
We can use the same format as:

https://github.com/scikit-learn/scikit-learn/blob/b78d62f9a3c0dd6af5ea3613a9364e228fef52c8/sklearn/model_selection/_search.py#L941-L948

Reply:
Thanks, I have updated the format for the input parameters according to _search.py